core/app/models/spree/variant.rb

```diff
@@ -40,8 +40,10 @@ class Variant < Spree::Base
     after_create :set_master_out_of_stock, unless: :is_master?
 
     after_touch :clear_in_stock_cache
+    after_save :update_counter_cache
 
-    scope :in_stock, -> { joins(:stock_items).where('count_on_hand > ? OR track_inventory = ?', 0, false) }
+    scope :in_stock, -> { joins(:stock_items).
+                          where("spree_stock_items.count_on_hand > ? OR track_inventory = ?", 0, false) }
```

Comment:
Line is too long. [105/80]<br>Expression at 46, 105 should be on its own line.<br>Use 2 (not 22) spaces for indenting an expression spanning multiple lines.